samples/batch/csharp/program.cs

```diff
@@ -6,136 +6,155 @@
+            var client = BatchClient.CreateApiV3Client(SubscriptionKey, $"{Region}.api.cognitive.microsoft.com");
 
             Console.WriteLine("Deleting all existing completed transcriptions.");
-            // delete all pre-existing completed transcriptions. If transcriptions are still running or not started, they will not be deleted
-            foreach (var item in transcriptions)
+
+            // get all transcriptions for the subscription
+            PaginatedTranscriptions paginatedTranscriptions = null;
+            do
             {
-                // delete a transcription
-                await client.DeleteTranscriptionAsync(item.Id).ConfigureAwait(false);
+                if (paginatedTranscriptions == null)
+                {
+                    paginatedTranscriptions = await client.GetTranscriptionsAsync().ConfigureAwait(false);
+                }
+                else
+                {
+                    paginatedTranscriptions = await client.GetTranscriptionsAsync(paginatedTranscriptions.NextLink).ConfigureAwait(false);
+                }
+
+                // delete all pre-existing completed transcriptions. If transcriptions are still running or not started, they will not be deleted
+                foreach (var transcriptionToDelete in paginatedTranscriptions.Values)
+                {
+                    // delete a transcription
+                    await client.DeleteTranscriptionAsync(transcriptionToDelete.Self).ConfigureAwait(false);
+                    Console.WriteLine($"Deleted transcription {transcriptionToDelete.Self}");
+                }
             }
+            while (paginatedTranscriptions.NextLink != null);
 
-            var transcriptionLocation = await client.PostTranscriptionAsync(Name, Description, Locale, new Uri(RecordingsBlobUri), modelList).ConfigureAwait(false);
-            var thisTranscriptionGuid = new Guid(transcriptionLocation.ToString().Split('/').LastOrDefault());
+            // <transcriptiondefinition>
+            var newTranscription = new Transcription
+            {
+                DisplayName = DisplayName, 
+                Locale = Locale, 
+                ContentUrls = new[] { RecordingsBlobUri },
+                //ContentContainerUrl = ContentAzureBlobContainer,
+                Model = CustomModel,
+                Properties = new TranscriptionProperties
+                {
+                    IsWordLevelTimestampsEnabled = true,
+                    TimeToLive = TimeSpan.FromDays(1)
+                }
+            };
 
-            // get the transcription Id from the location URI
-            var createdTranscriptions = new List<Guid>();
-            createdTranscriptions.Add(thisTranscriptionGuid);
+            newTranscription = await client.PostTranscriptionAsync(newTranscription).ConfigureAwait(false);
+            Console.WriteLine($"Created transcription {newTranscription.Self}");
+            // </transcriptiondefinition>
```

Comment:
> </transcriptiondefinition> [](start = 14, length = 27)

the section is not used in any other places. remove it?

Reply:
used in docs